addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/DirectoryBrowsingScanRule.java

```diff
@@ -137,7 +137,8 @@ public void scan() {
         if (result) {
             newAlert()
                     .setConfidence(confidence)
-                    .setAttack(evidence.toString())
+                    .setAttack(msg.getRequestHeader().getPrimeHeader())
```

Comment:
I'm not really sure this is helpful. The alert is already associated with a URL and the URL hasn't been manipulated here or anything.

The prime header on a request is like `GET https://googleads.g.doubleclick.net/pagead/drt/si HTTP/1.1`

Reply:
I have the assumption that the alert attack parameter represents what we did and the evidence is something in the response that is an evidence that the vulnerability exists. If this assumption is correct the attack is requesting a path and the evidence is a response with directory listing. My assumption may be wrong.

Reply:
Then make the "attack" be the URL instead of the whole request prime header?
`msg.getRequestHeader().getURI().toString()`

Reply:
It makes, but I think it is the same as the prime header. Should I change it?

Reply:
Using the URI is just a bit less data and is more specific (the method and protocol aren't specific to the attack).

Prime: `GET https://googleads.g.doubleclick.net/pagead/drt/si HTTP/1.1`
vs
URI: `https://googleads.g.doubleclick.net/pagead/drt/si`